verilator-config.cmake.in

```diff
@@ -77,12 +77,6 @@ if (NOT CMAKE_CXX_COMPILER_ID MATCHES MSVC)
   endif()
 endif()
 
-define_property(TARGET
-  PROPERTY VERILATOR_THREADED
-  BRIEF_DOCS "Verilator multithreading enabled"
-  FULL_DOCS "Verilator multithreading enabled"
-)
-
```

Comment:
Also remove where this is used?

Reply:
I wonder if we should leave this and document that it does nothing? (Even if set false). Otherwise cmake files that are setting it will break, which seems unfriendly.

Reply:
To my knowledge, ``define_property`` is not strictly needed, you can use ``set_property`` on property that hasn't been defined without any error, but I couldn't find any information about this in the official documentation, only it is mention here in the discussion: https://gitlab.kitware.com/cmake/cmake/-/issues/19373#note_583175. I've tried to set property that wasn't defined before it and cmake didn't error out, so it looks like this information is still valid.

Anyway, I've restored this as it is probably still good to inform user that this property is now deprecated and ignored.